docs/data/data-grid/joy-ui/GridJoyUISlots.tsx

```diff
@@ -93,16 +93,29 @@ for (let i = 0; i < 20; i += 1) {
 }
 
 export default function GridJoyUISlots() {
+  const [rows, setRows] = React.useState<Row[]>([]);
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    setLoading(true);
+    const timeoutId = setTimeout(() => {
+      setRows(initialRows);
+      setLoading(false);
+    }, 800);
+    return () => clearTimeout(timeoutId);
+  }, []);
+
   return (
     <MaterialCssVarsProvider theme={{ [MATERIAL_THEME_ID]: materialTheme }}>
       <JoyCssVarsProvider>
-        <Box sx={{ height: 420, width: '100%' }}>
+        <Box sx={{ height: 400, width: '100%' }}>
           <DataGrid
             pagination
             slots={{
               ...unstable_joySlots,
               toolbar: GridToolbar,
             }}
+            loading={loading}
```

Comment:
@siriwatknp I've updated the demo to show a single grid in a loading state. What do you think?